packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -120,9 +119,8 @@ export default class LocusInfo extends ChildEmitter {
    */
   onFullLocus(locus, eventType) {
     if (!locus) {
-      console.log('locus object undefined');
+      LoggerProxy.logger.error('LocusInfo->onFullLocus#Locus object passed as argument was invalid, continuing.');
```

Comment:
Would something like this be considered `verbose`?

Reply:
anything that is routed through EventProxy will be considered verboseEvents, I don't have a general verbose config yet, just a general verboseEvents :) to keep it isolated to that. if we want to add a verboseErrors config we could do that.